spec/system/admin/admin_manages_pages_spec.rb

```diff
@@ -111,4 +112,37 @@
       end
     end
   end
+
+  describe "when there is a landing page" do
```

Comment:
It would be great if these could be turned into Cypress admin tests from the perspective of the user. I think this would require:

- adding a page or two to https://github.com/forem/forem/blob/main/spec/support/seeds/seeds_e2e.rb
- going through the various scenarios with Cypress, you can check https://github.com/forem/forem/tree/main/cypress/integration/adminFlows to get a sense of how they work and the doc for how to start https://docs.forem.com/tests/e2e-tests/

Again, let someone know if you need help :)

Reply:
I've added a Cypress Admin test for this work! The test itself isn't passing at this moment, but I'm working on it. 😊